core/pbcc_actions_api.h

```diff
@@ -0,0 +1,109 @@
+/* -*- c-file-style:"stroustrup"; indent-tabs-mode: nil -*- */
+#if !defined INC_PBCC_ACTIONS_API
+#define INC_PBCC_ACTIONS_API
+
+#include "pubnub_api_types.h"
+#include "pubnub_memory_block.h"
+
+struct pbcc_context;
+
+
+/** @file pbcc_actions_api.h
```

Comment:
Lol, looks like Message Actions in this. I suppose it's fine, but you can remove it if you're not done. I'm ok either way.